Commands/Fields/GetField.cs

```diff
@@ -9,12 +9,16 @@ namespace SharePointPnP.PowerShell.Commands.Fields
     [Cmdlet(VerbsCommon.Get, "SPOField")]
     [CmdletHelp("Returns a field from a list or site",
         Category = CmdletHelpCategory.Fields)]
+    [CmdletExample(
+        Code = @"PS:> Get-SPOField -List ""Demo list"" -Identity ""Speakers""",
+        Remarks = @"Gets the speakers field from the list Demo list",
+        SortOrder = 1)]
```

Comment:
Maybe good to also specify an example with only

```powershell
Get-SPOField 
```

Specify the cmdlet without parameters willl make it return all site columns.